frontend/Makefile

```diff
@@ -46,17 +46,15 @@ clean: ## Purges node_modules
 	@rm -rf node_modules
 	@rm -rf $(BUILD_DIR)
 
-node_modules: package.json
-	@npm ci
-
-install: node_modules ## Installs NPM dependencies
+install: ## Installs NPM dependencies
 	@echo "$(P) Installing dependencies..."
+	@npm ci
 
-build: install ## Builds the frontend
+build: ## Builds the frontend
 	@echo "$(P) Building in release mode..."
 	@npm run build
 
-run: install ## Serves the frontend via webpack-dev-server
+run: ## Serves the frontend via webpack-dev-server
```

Comment:
do not force a slow "npm ci" operation before building or running locally. Most devs are on top of their dependencies anyways